src/components/wiki/WikiPage/WikiPage.tsx

```diff
@@ -0,0 +1,47 @@
+import { MainLayout } from '@/components/MainLayout/MainLayout';
+import { MainNavigation } from '@/components/MainNavigation/MainNavigation';
+import { Heading } from '@chakra-ui/react';
+import React, { useMemo } from 'react';
+import ReadMore from '../ReadMore';
+import Tip from '../Tip';
+import { Topics } from '../Topic';
+
+interface WikiPageProps {
+	title: string;
+	content: Array<{
+		title: string;
+		description: string;
+	}>;
+	topic: Topics;
+}
+
+const WikiPage: React.FC<WikiPageProps> = ({ title, content, topic }) => {
+	const accentColor = useMemo(() => {
+		switch (topic) {
+			case Topics.Requirements:
+				return '#4042E2';
+			case Topics.Design:
+				return '#FF8077';
+			case Topics.Deployment:
+				return '#F29A33';
+			case Topics.Testing:
+				return '#A13976';
+			case Topics.Development:
+				return '#7D97F4';
+		}
+	}, [topic]);
```

Comment:
This is not really a good pattern. If we would want to add a new chapter, we would need to create a page and update this piece of code; but if we make `accentColor` a prop, then only a page would be created.

Also, with your way, we are shipping extra code that might not be used; with proposed solution.